### PR TITLE
fix: multiline and long commit message break formatting

### PR DESCRIPTION
**Pull request by radiantspace**
Context: https://github.slack.com/archives/C016R13H5FV/p1598879831070700

Screen Shot 2020-08-31 at 18 59 10

### CODE REVIEW

**Review by Soph — APPROVED** [review]

File: src/utils.ts [hunk 1]
```diff
@@ -64,7 +64,8 @@ export async function postSlackNotification (
     if (payloadForPushes?.compare !== undefined) {
       const beforeSha = payloadForPushes.before.slice(0, 7)
       const afterShaMessage = payloadForPushes.head_commit.message ?? ''
-      commitText = `<${payloadForPushes.compare}|${beforeSha} ⇢ ${afterSha} ${afterShaMessage}>`
+      const shortShaMessage = trimEllipsis(afterShaMessage.replace(/(\r\n|\n|\r).*$/gm, ''), 60) // keep only some first symbols of the first line
```
Comment (lumaxis) [inline src/utils.ts:67]: How did you pick 60? I think 80 characters is a common recommended git commit message limit but not completely sure 🤔

File: src/utils.ts [hunk 2]
```diff
@@ -84,3 +85,7 @@ export async function postSlackNotification (
     core.error(error)
   }
 }
+
+function trimEllipsis (str: string, length: number): string {
+  return str.length > length ? `${str.substring(0, length)}...` : str
```
Comment (lumaxis) [inline src/utils.ts:90]:
```suggestion
  return (str.length > length) ? `${str.substring(0, length)}...` : str
```

File: src/utils.ts (hunk 1 above)
Comment (radiantspace) [reply src/utils.ts:67]: So google says it’s 72 chars, but I thought of having shorter one to reduce chat spam. We can adjust that as we go if that will be annoying.

**Review by gordey4doronin — COMMENTED** [review]
👍

**Pull request merged**